assets/css/style.css

```diff
@@ -43,4 +63,27 @@
 .brand .name {
     font-size: 24px;
 }
-/* End of Brand */
\ No newline at end of file
+/* End of Brand */
+
+/* Navbar */
+.nav {
```

Comment:
i think nav was meanness in our app.
There are some better name we can consider., such as: main-navigation, main-menu, ....
as main-menu, it means that the main navigation of the app.